t/t2403-job-exec-conf.t

```diff
@@ -93,60 +99,65 @@ test_expect_success 'job-exec: can specify exec service on cmdline' '
 '
 test_expect_success 'job-exec: exec service can be set in exec conf' '
 	name=exec-service &&
-	cat <<-EOF > ${name}.toml &&
+	mkdir ${name} &&
+	cat <<-EOF > ${name}/${name}.toml &&
 	[exec]
 	service = "bar"
 	EOF
-	flux start -o,--config-path=${name}.toml -s1 \
+	flux start -o,--config-path=${name}/${name}.toml -s1 \
 		flux module stats -p impl.bulk-exec.config.exec_service job-exec > ${name}.out 2>&1 &&
 	grep "bar" ${name}.out
 '
 test_expect_success 'job-exec: exec service override can be set in exec conf' '
 	name=exec-service-override &&
-	cat <<-EOF > ${name}.toml &&
+	mkdir ${name} &&
+	cat <<-EOF > ${name}/${name}.toml &&
 	[exec]
 	service = "bar"
 	service-override = true
 	EOF
-	flux start -o,--config-path=${name}.toml -s1 \
+	flux start -o,--config-path=${name}/${name}.toml -s1 \
 		flux module stats -p impl.bulk-exec.config.exec_service_override job-exec > ${name}.out 2>&1 &&
 	val=$(cat ${name}.out) &&
 	test $val -eq 1
 '
 test_expect_success 'job-exec: sdexex properties can be set in exec conf' '
 	name=sdexec-properties &&
-	cat <<-EOF > ${name}.toml &&
+	mkdir ${name} &&
+	cat <<-EOF > ${name}/${name}.toml &&
 	[exec]
 	service = "sdexec"
 	[exec.sdexec-properties]
 	MemoryHigh = "200M"
 	MemoryMax = "100M"
 	EOF
-	flux start -o,--config-path=${name}.toml -s1 \
+	flux start -o,--config-path=${name}/${name}.toml -s1 \
 		flux module stats -p impl.bulk-exec.config.sdexec_properties job-exec > ${name}.out 2>&1 &&
 	jq -e ".MemoryHigh == \"200M\"" < ${name}.out &&
 	jq -e ".MemoryMax == \"100M\"" < ${name}.out
 '
 test_expect_success 'job-exec: bad sdexec properties causes module failure (type 1)' '
 	name=bad-sdexec-properties1 &&
-	cat <<-EOF > ${name}.toml &&
+	mkdir ${name} &&
+	cat <<-EOF > ${name}/${name}.toml &&
 	[exec]
 	service = "sdexec"
 	sdexec-properties = 42
 	EOF
-	test_must_fail flux start -o,--config-path=${name}.toml -s1 \
+	test_must_fail flux start -o,--config-path=${name}/${name}.toml -s1 \
 		flux dmesg > ${name}.log 2>&1 &&
 	grep "exec.sdexec-properties is not a table" ${name}.log
 '
 test_expect_success 'job-exec: bad sdexec properties causes module failure (type 2)' '
 	name=bad-sdexec-properties2 &&
-	cat <<-EOF > ${name}.toml &&
+	mkdir ${name} &&
+	cat <<-EOF > ${name}/${name}.toml &&
 	[exec]
 	service = "sdexec"
 	[exec.sdexec-properties]
 	MemoryHigh = 42
 	EOF
-	test_must_fail flux start -o,--config-path=${name}.toml -s1 \
+	test_must_fail flux start -o,--config-path=${name}/${name}.toml -s1 \
 		flux dmesg > ${name}.log 2>&1 &&
```

Comment:
yeah, it was necessary, although maybe the commit message needs more information.  By putting the .toml files in `pwd` directory, other tests `flux config reload` might pick this toml file up by accident.  I could delete the toml file instead, but I thought we'd want to keep as many of them around as we can for debugging.

Reply:
Ah well maybe it would be clearer to point FLUX_CONF_DIR=$(pwd)/conf.d and keep that config separate from the single-config file tests rather than move each single config test file into its own directory?